test/Microsoft.NET.Sdk.Functions.Generator.Tests/FunctionJsonConverterTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using FluentAssertions;
+using MakeFunctionJson;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.ServiceBus;
+using Microsoft.ServiceBus.Messaging;
+using Microsoft.WindowsAzure.Storage.Queue;
+using Xunit;
+
+namespace Microsoft.NET.Sdk.Functions.Test
+{
+    public class FunctionJsonConverterTests
+    {
+        public class FunctionsClass
+        {
+            [FunctionName("MyHttpTrigger")]
+            public static void Run1([HttpTrigger] HttpRequestMessage request) { }
+            
+            [FunctionName("MyBlobTrigger")]
+            public static void Run2([BlobTrigger("blob.txt")] string blobContent) { }
+            
+            [FunctionName("MyQueueTrigger")]
+            public static void Run3([QueueTrigger("queue")] CloudQueue queue) { }
+            
+            [FunctionName("MyEventHubTrigger")]
+            public static void Run4([EventHubTrigger("hub")] EventData message) { }
+            
+            [FunctionName("MyTimerTrigger")]
+            public static void Run5([TimerTrigger("00:30:00")] TimerInfo timer) { }
+            
+            [FunctionName("MyServiceBusTrigger")]
+            public static void Run6([ServiceBusTrigger("queue")] string message) { }
+                        
+            [FunctionName("MyManualTrigger"), NoAutomaticTrigger]
+            public static void Run7(string input) { }
+            
+            [FunctionName("MyManualTriggerWithoutParameters"), NoAutomaticTrigger]
+            public static void Run8() { }
```

Comment:
What function.json gets generated in this case?

Reply:
```json
{
  "generatedBy": "Microsoft.NET.Sdk.Functions.Generator-1.0.5",
  "configurationSource": "attributes",
  "bindings": [
    {
      "type": "manualTrigger",
      "direction": "in"
    }
  ],
  "disabled": false,
  "scriptFile": "../../../test/Microsoft.NET.Sdk.Functions.Generator.Tests/bin/Debug/Microsoft.NET.Sdk.Functions.Test.dll",
  "entryPoint": "Microsoft.NET.Sdk.Functions.Test.FunctionJsonConverterTests+FunctionsClass.Run8"
}
```